cli/zoo.py

```diff
@@ -76,5 +76,55 @@ def manager(port):
     zoo_manager.serve(port)
 
 
+@zoo_cli.command(
+    name="install",
+    help="Attempt to install the specified agents from the given paths/url",
+)
+@click.argument(
+    "agent_paths",
+    type=click.Path(exists=True),
+    metavar="<script>",
+    nargs=-1,
+    required=True,
+)
+def install_agents(agent_paths):
+    if not agent_paths:
```

Comment:
Why do `if not agent_paths:` when `agent_paths` has `required=True`?

Reply:
Apparently even is we set `required=True`, the command runs without it so I just added a safety net to be safe.